src/linear_algebra/dual.lean

```diff
@@ -570,6 +583,24 @@ begin
     exact is_basis.to_dual_equiv _ hB.1 },
 end
 
```

Comment:
There's a lot of long lines containing `finite_dimensional.findim`. Let's put this line here (or at the beginning of the file), so we can write just `findim`. 
```suggestion
open finite_dimensional
```